types/javascript-time-ago/OTHER_FILES.txt

```diff
@@ -0,0 +1,131 @@
+locale/lb/index.d.ts
```

Comment:
Only 3 are ever tested, which is reasonable since all the files are nearly identical.